upload_form.py

```diff
@@ -345,15 +345,15 @@ def set_columns(filename, fileformat=None):
 
     append_table(merged_table, table)
     Table.write(merged_table, merged_table_name, format='ascii.ipac')
-
+    
     username = column_data.get('Username')['Name']
     branch,timestamp = commit_change_to_database(username)
     time.sleep(2)
-    # Adding raw file to uploads
-    #dummy = commit_change_to_database(username, tablename=filename, workingdir='uploads/',database='upl')
-    #time.sleep(2)
     pull_request(branch, username, timestamp)
-    #pull_request(branch, username, timestamp, database='uploads')
+    # Adding raw file to uploads
+    dummy = commit_change_to_database(username, tablename=filename, workingdir='uploads/',database='upl',branch=branch)
```

Comment:
that was my first guess, but it does not work. In the error message it said that the db name was not what it expected: it said that it should have been 'upl' instead. In this way it works though.
Do you know how this name = os.path.split(check_upstream)[1][:-5] turns out to be upl instead of uploads?